sdk/python/tests/compiler/testdata/affinity.yaml

```diff
@@ -1,17 +1,3 @@
-# Copyright 2020 kubeflow.org
```

Comment:
add back the license header. flipping on the `GENERATE_GOLDEN_YAML` flag should do that in the compiler unit test suite

Reply:
Cool, put it in there!